tests/unit/core/test_deploy_export.py

```diff
@@ -347,9 +359,12 @@ def test_quartz_encoder(self):
         jasper_encoder = nemo_asr.JasperEncoder(
             conv_mask=False,
             feat_in=quartz_model_definition['AudioToMelSpectrogramPreprocessor']['features'],
-            **quartz_model_definition['JasperEncoder']
+            **quartz_model_definition['JasperEncoder'],
```

Comment:
The same here...

Moreover, after the refactor those two tests could be then unified into 1 test, parametrized additionally by the configuration file! :D